docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        mapped_genes = gtf[gtf[gtf_col].isin(adata.var_names)].drop_duplicates(
</pre><p>minor comment here: </p><p> </p><p>you can do <code>gtf[lambda x: x[gtf_col].isin(...)]</code> instead of repeating the name of the data frame. Saves some typing for longer variable names and, most importantly, prevents copy&amp;paste errors. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>